sympy/printing/pycode.py

```diff
@@ -995,6 +995,23 @@ def _print_uppergamma(self, expr):
             self._print(expr.args[0]),
             self._print(expr.args[1]))
 
+    def _print_betainc(self, expr):
+        return "({0}({2}, {3}, {5}) - {0}({2}, {3}, {4})) * {1}({2}, {3})".format(
+            self._module_format('scipy.special.betainc'),
+            self._module_format('scipy.special.beta'),
```

Comment:
Using an f-string might make this a bit clearer

Reply:
Something like
```python
def _print_betainc(self, expr):
    sp_betainc = self._module_format('scipy.special.betainc')
    sp_beta = self._module_format('scipy.special.beta')
    args = [self._print(arg) for arg in expr.args]
    return f"({sp_betainc}({args[0]}, {args[1]}, {args[3]}) ...
```
Better if we can give reasonable names to the args.

Reply:
All the printers seem to use the same type of syntax. Would it be okay to make this different? As for args, we could make it `print_args`

Reply:
I've changed it to what you suggested.